.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -1,18 +1,39 @@
+Fixes: *<insert link to GitHub issue here>*
+
 ### What was the problem/requirement? (What/Why)
 
 ### What was the solution? (How)
 
 ### What is the impact of this change?
 
 ### How was this change tested?
-- Have you run `hatch run test` ?
 
-- Have you run `hatch run integ:test` ? See `DEVELOPMENT.md` on "Run integration tests"
+See `DEVELOPMENT.md` for information on testing.
+
+- Have you run the unit tests (`hatch run test`) ?
+- Have you run the integration tests (`hatch run integ:test`) ?
+
+- Have you made changes to the `download` or `asset_sync` modules? If so, then it is highly recommended
```

Comment:
Can we possibly link to the section on how to run these, or a blurb of what you're supposed to run specifically? Seeing as you have `hatch run test` /etc. for the other two steps

Reply:
I can link to the relevant section of the developer doc, sure. Good suggestion!